spec/system/candidate_interface/candidate_signing_in_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+
+describe 'A candidate signing in' do
+  context 'who successfully signs in' do
```

Comment:
This context doesn't necessarily apply to the `sees the sign in page` spec. Could the `before` live at the top level, and then this context wrap the `sees the "Check your email" page`?

Reply:
I agree! It should be good now:
```A candidate signing in
  sees the sign in page
  who successfully signs in
    sees the "Check your email" page
```